packages/admin/test/e2e/cli.feature.ts

```diff
@@ -0,0 +1,496 @@
+      const requesterIndex = await admin.createRequester(airnodeRrp, alice.address);
+      const clientAddress = bob.address;
+      airnodeRrp = airnodeRrp.connect(alice);
+      await admin.endorseClient(airnodeRrp, requesterIndex, clientAddress);
+
+      const isEndorsed = () =>
+        admin.requesterIndexToClientAddressToEndorsementStatus(airnodeRrp, requesterIndex, clientAddress);
+
+      expect(await isEndorsed()).toBe(true);
+      const out = execCommand(
+        'unendorse-client',
+        ['--mnemonic', mnemonic],
+        ['--derivationPath', aliceDerivationPath],
+        ['--providerUrl', PROVIDER_URL],
+        ['--airnodeRrp', airnodeRrp.address],
+        ['--requesterIndex', requesterIndex],
+        ['--clientAddress', clientAddress]
+      );
+      expect(out).toBe(`Client address: ${clientAddress}`);
+      expect(await isEndorsed()).toBe(false);
+    });
+
+    it('check endoresement status', async () => {
+      const requesterIndex = await admin.createRequester(airnodeRrp, alice.address);
+      const clientAddress = bob.address;
+
+      const checkEndorsement = () =>
+        execCommand(
+          'get-endorsement-status',
+          ['--providerUrl', PROVIDER_URL],
+          ['--airnodeRrp', airnodeRrp.address],
+          ['--requesterIndex', requesterIndex],
+          ['--clientAddress', clientAddress]
+        );
+
+      expect(checkEndorsement()).toBe('Endorsment status: false');
+      airnodeRrp = airnodeRrp.connect(alice);
+      await admin.endorseClient(airnodeRrp, requesterIndex, clientAddress);
+      expect(checkEndorsement()).toBe('Endorsment status: true');
+    });
+  });
+
+  describe('templates', () => {
+    const createTemplate = (fileName: string) =>
+      execCommand(
+        'create-template',
+        ['--mnemonic', mnemonic],
+        ['--derivationPath', aliceDerivationPath],
+        ['--providerUrl', PROVIDER_URL],
+        ['--airnodeRrp', airnodeRrp.address],
+        ['--templateFilePath', `${__dirname}/../fixtures/${fileName}`]
+      );
+
+    it('can create template', async () => {
+      const out = createTemplate('template.json');
+      expect(out).toMatch(new RegExp('Template ID: 0x\\w+'));
+    });
+
+    it('errors out on invalid template file', () => {
+      expect(() => createTemplate(`non-existent-file.json`)).toThrow('ENOENT: no such file or directory');
+    });
+
+    it('can get template', () => {
+      const templateId = createTemplate('template.json').split('Template ID: ')[1];
+
+      const out = execCommand(
+        'get-template',
+        ['--providerUrl', PROVIDER_URL],
+        ['--airnodeRrp', airnodeRrp.address],
+        ['--templateId', templateId]
+      );
+
+      expect(JSON.parse(out)).toEqual({
+        airnodeId: '0x15e7097beac1fd23c0d1e3f5a882a6f99ecbcf2e0c1011d1bd43707c6c0ec717',
+        endpointId: '0x2605589dfc93c8f9c35eecdfe1e666c2193df30a8b13e1e0dd72941f59f9064c',
+        parameters: expect.anything(), // parameters are encoded
```

Comment:
Should be `0x31537500000000000000000000000000000000000000000000000000000000006e616d653100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a06e616d653200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000003e8000000000000000000000000000000000000000000000000000000000000000676616c7565310000000000000000000000000000000000000000000000000000`

Reply:
I did it on purpose. There is not much value testing that as those test should mainly verify the shape of the data. Whether the paramers endoding works should be covered elsewhere (in protocol pckg).

But that's just my preference, so I'll use your suggestion.